components/_expanding-components/section-initialization.js

```diff
@@ -0,0 +1,4 @@
+import { UncheckRadioButton } from './uncheck-radio-button.js'
+
```

Comment:
I guess you will run into trouble on the docs if this has not been added to window object? 

and you should check for it aswell if(expandingSections )

Reply:
The documentation works as expected, so I don't think its an issue. But maybe we should look at this one together